### PR TITLE
2 names and 1 ext added

### PR DESCRIPTION
**Pull request by lcnittl**

### CODE REVIEW

File: identify/extensions.py [hunk 1]
```diff
@@ -183,11 +183,12 @@
     'vue': {'text', 'vue'},
     'war': {'binary', 'zip', 'jar'},
     'wav': {'binary', 'audio', 'wav'},
-    'wkt': {'text', 'wkt'},
     'whl': {'binary', 'wheel', 'zip'},
+    'wkt': {'text', 'wkt'},
     'woff': {'binary', 'woff'},
     'woff2': {'binary', 'woff2'},
     'wsgi': {'text', 'wsgi', 'python'},
+    'xhtml': {'text', 'xml', 'xhtml'},
```
Comment (lcnittl) [inline identify/extensions.py:191]: Should `xhtml` have `html` type as well?

**Comment by asottile** [comment]
can you sort this in a separate PR, (some of them probably shouldn't be sorted, for instance `adoc` and `asciidoc` are next to each other because the values are the same) -- as it is now, it's really difficult to see what was added in this PR

**Comment by lcnittl** [comment]
Sure, done

**Comment by lcnittl** [comment]
> for instance `adoc` and `asciidoc` are next to each other because the values are the same

Best use case for #134 :rofl:

File: identify/extensions.py (hunk 1 above)
Comment (asottile) [reply identify/extensions.py:191]: yeah probably, sorry missed this first time around!

**Review by asottile — APPROVED** [review]

File: identify/extensions.py (hunk 1 above)
Comment (lcnittl) [reply identify/extensions.py:191]: Sorry, saw it just now!

Comment (lcnittl) [reply identify/extensions.py:191]: Shall I add it still in another PR?

Comment (asottile) [reply identify/extensions.py:191]: up to you!

**Pull request merged**